htdocs/wp-content/install.php

```diff
@@ -107,216 +119,434 @@ function wp_install( $blog_title, $user_name, $user_email, $public, $deprecated
+
+        $wpdb->insert(
+            $wpdb->posts, array(
+                'post_author'           => $user_id,
+                'post_date'             => $now,
+                'post_date_gmt'         => $now_gmt,
+                'post_content'          => $privacy_policy_content,
+                'post_excerpt'          => '',
+                'comment_status'        => 'closed',
+                'post_title'            => __( 'Privacy Policy' ),
+                /* translators: Privacy Policy page slug */
+                'post_name'             => __( 'privacy-policy' ),
+                'post_modified'         => $now,
+                'post_modified_gmt'     => $now_gmt,
+                'guid'                  => $privacy_policy_guid,
+                'post_type'             => 'page',
+                'post_status'           => 'draft',
+                'to_ping'               => '',
+                'pinged'                => '',
+                'post_content_filtered' => '',
+            )
+        );
+        $wpdb->insert(
+            $wpdb->postmeta, array(
+                'post_id'    => 2,
+                'meta_key'   => '_wp_page_template',
+                'meta_value' => 'default',
+            )
+        );
+        update_option( 'wp_page_for_privacy_policy', 2 );
+    }
+
+    // Seravo: Don't setup standard widgets to make the install cleaner
+    // Set up default widgets for default theme.
+
+    if ( ! is_multisite() )
+        update_user_meta( $user_id, 'show_welcome_panel', 1 );
+    elseif ( ! is_super_admin( $user_id ) && ! metadata_exists( 'user', $user_id, 'show_welcome_panel' ) )
+        update_user_meta( $user_id, 'show_welcome_panel', 2 );
+
+    if ( is_multisite() ) {
+        // Flush rules to pick up the new page.
+        $wp_rewrite->init();
+        $wp_rewrite->flush_rules();
+
+        $user = new WP_User($user_id);
+        $wpdb->update( $wpdb->options, array('option_value' => $user->user_email), array('option_name' => 'admin_email') );
+
+        // Remove all perms except for the login user.
+        $wpdb->query( $wpdb->prepare("DELETE FROM $wpdb->usermeta WHERE user_id != %d AND meta_key = %s", $user_id, $table_prefix.'user_level') );
+        $wpdb->query( $wpdb->prepare("DELETE FROM $wpdb->usermeta WHERE user_id != %d AND meta_key = %s", $user_id, $table_prefix.'capabilities') );
+
+        // Delete any caps that snuck into the previously active blog. (Hardcoded to blog 1 for now.) TODO: Get previous_blog_id.
+        if ( !is_super_admin( $user_id ) && $user_id != 1 )
+            $wpdb->delete( $wpdb->usermeta, array( 'user_id' => $user_id , 'meta_key' => $wpdb->base_prefix.'1_capabilities' ) );
+    }
+
+    /**
+     * Seravo specific option changes that are typically not part of core WordPress wp_install_defaults function
+     */
+
+    /** @see wp-admin/options-general.php */
+    // Remove standard blog description
+    update_option( 'blogdescription', '' );
```

Comment:
Why remove? You can just overwrite it with custom content at any give point later on.

Reply:
Apparently the blog description is set in schema.php before this is run, so the way we set WP_LANG here in install.php does not work on that content.

This means that if we don't update this here the site will have the English standard blog description even if we have WP_LANG set to eg sv_SE.

And since I don't think it makes sense to have a Seravo specific blog description I instead decided to remove it.